modules/es-extensions/store/asset/schema/asset.js

```diff
@@ -158,8 +158,29 @@ asset.manager = function(ctx) {
             return asset.name;
         },
         getVersion: function(asset) {
+            if (!asset.attributes["version"]) {
+                var subPaths = asset.path.split('/');
+                asset.version = subPaths[subPaths.length - 2];
```

Comment:
we can write this as follows in order to be consistent with the rest of the code base:

asset.attributes.version = asset.version